### ISSUE TITLE
[MVT] Localization : red ballon not shown

### ISSUE THREAD

**Issue by davidoesch** [geoadmin/mf-geoadmin3#4606]
_OS/Device_
ALL OS
ALL Devices

_Steps to reproduce_

1.        Open application  https://mf-geoadmin3.int.bgdi.ch/mvt/index.html?
2.        Enter ""Genè"" in localization textbox and select ""Genève"" /same for ""Längmatt 19 3608 Thun""
3.        Enter 200000 600000 in localization textbox
4.        Enter 7.5 47 in localization textbox
5. Enter ""evech"" in localization textbox and select ""Rue de l'Évêché 1 1204 Genève""
6.        Write ""?swisssearch=payerne"" in the URL and then enter"

_result_
no red ballon on result

_expected result_
red ballon on position

_Workaround:_
none

**Comment by loicgasser** [comment]
1. NA
2. This is correct and incorrect in prod. we only add a marker for the types address, parcel and gazeteer. See https://github.com/geoadmin/mf-geoadmin3/blob/master/src/components/search/SearchTypesDirectives.js#L29 for add overlay.
3. Fixed
4. FIxed. `7.5 47` is outside Switzerland, try `47 7.5` instead. I fixed it to have the same behavior than in prod, but that means we cannot input coords outside CH.
5. works
6. works search suggestions appear
